projects/dsp-ui/src/lib/viewer/values/link-value/link-value.component.ts

```diff
@@ -65,10 +76,11 @@ export class LinkValueComponent extends BaseValueComponent implements OnInit, On
     searchByLabel(searchTerm: string) {
         // at least 3 characters are required
         if (typeof searchTerm === 'string' && searchTerm.length >= 3) {
-            this._dspApiConnection.v2.search.doSearchByLabel(searchTerm, 0, { limitToResourceClass: this.restrictToResourceClass }).subscribe(
-                (response: ReadResourceSequence) => {
-                    this.resources = response.resources;
-                });
+            this._dspApiConnection.v2.search.doSearchByLabel(
```

Comment:
reformatting?

Reply:
yeah, it was exceeding the character limit